helpers/any.js

```diff
@@ -15,25 +14,48 @@ const factory = () => {
         const opts = args.pop();
         const predicate = opts.hash;
 
-        if (!_.isEmpty(predicate)) {
+        if (!common.isEmptyObject(predicate)) {
             // With options hash, we check the contents of first argument
-            any = _.any(args[0], predicate);
+            const predicateKeys = Object.keys(predicate);
+            const arg = args[0];
+            if (Array.isArray(arg)) {
+                for (let i = 0; i < arg.length; i++) {
+                    const element = arg[i];
+                    for (let j = 0; j < predicateKeys.length; j++) {
```

Comment:
This inner portion is common to both cases so let's pull it out into a common function.

Reply:
fixed.